packages/yoroi-extension/app/containers/NavBarContainer.js

```diff
@@ -115,8 +105,12 @@ export default class NavBarContainer extends Component<Props> {
           return <NoWalletsDropdown />;
         }
 
-        const balance: ?MultiToken = this.props.stores.transactions.getBalance(publicDeriver);
-        const rewards: MultiToken = this.props.stores.delegation.getRewardBalanceOrZero(publicDeriver);
+        const balance: ?MultiToken = this.props.stores.transactions.getBalance(publicDeriver.publicDeriverId);
+        const rewards: MultiToken = this.props.stores.delegation.getRewardBalanceOrZero(
+          publicDeriver.publicDeriverId,
+          publicDeriver.networkId,
+          publicDeriver.defaultTokenId
+        );
```

Comment:
Kinda weird that this function changed from just accepting the wallet type to accepting multiple fields all of which are coming from the wallet type. Might make sense to change it to just accept the wallet type and then retrieve the required fields internally, unless there's a good reason not to.